db/migrate/20200723212151_create_trial_attribute_settings.rb

```diff
@@ -0,0 +1,36 @@
+class CreateTrialAttributeSettings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :trial_attribute_settings do |t|
+      t.integer :system_info_id
+      t.string :attribute_name
+      t.string :attribute_key
+      t.string :attribute_label
+      t.boolean :display_label_on_list, default: true
+      t.boolean :display_attribute_on_list, default: true
+      t.boolean :display_attribute_if_null_on_list, default: true
+      t.boolean :display_label_on_show, default: true
+      t.boolean :display_attribute_on_show, default: true
+      t.boolean :display_attribute_if_null_on_show, default: true
+    end
+
+    #seed data 
+    system = SystemInfo.first
```

Comment:
This breaks for new checkouts (or when you reset the DB like I did). This project has to run migrations (versus loading the schema file) because the views are not present in the schema file.

Generally, I think this kind of initial data is OK to include in migrations. It makes for a good upgrade experience because folks with existing installations don't have to take extra steps. Maybe we can stick this data into a file that can be referenced from both here and the seeds? If we keep these here in the migration, we should move it out of the `change` method or stick it in an `up_only` block so it only runs when migrating up.